src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs

```diff
@@ -608,9 +626,9 @@ private string PreProcessAuthScope()
             return mappedScope;
         }
 
-        private bool IsUsingInteractiveAuthentication()
+        private bool IsBrowserPopUpInteractiveAuthentication()
         {
-            return ParameterSetName == UserParameterSet && UseDeviceAuthentication == false;
+            return ParameterSetName.Equals(UserParameterSet) && UseDeviceAuthentication.IsPresent == false;
```

Comment:
FYI `UseDeviceAuthentication` and `UseDeviceAuthentication.IsPresent` have the exact same effect